react-native-libraries.json

```
@@ -14363,5 +14363,25 @@
     "examples": [
       "https://github.com/mahdidavoodi7/expo-image-compare/tree/main/example"
     ]
+  },
+  {
+    "githubUrl": "https://github.com/Hiroenzo/react-native-mqtt-clients",
+    "npmPkg": "https://www.npmjs.com/package/react-native-mqtt-clients",
+    "examples": [
+      "https://github.com/Hiroenzo/react-native-mqtt-clients/tree/main/example"
+    ],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "visionos": false,
+    "expoGo": false,
+    "fireos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```

Hey @mrzslr, thanks for adding a new package data to the directory! 👍 

Let's fix and clean the entry up before shipping. This also should fix the CI:

```suggestion
    "githubUrl": "https://github.com/Hiroenzo/react-native-mqtt-clients",
    "examples": [
      "https://github.com/Hiroenzo/react-native-mqtt-clients/tree/main/example"
    ],
    "ios": true,
    "android": true
```

Explanation of the changes:
* `npmPkg` expects npm registry package name, not URL, it is also automatically backfilled for most of the packages, and main purpose of that field is for scoped packages (i.e. `@expo/fingerprint`),
* all falsy values can and should be skipped.

See the adding library section in our Readme to learn more:
* https://github.com/react-native-community/directory?tab=readme-ov-file#new-library-entry-template